src/test/java/core/basesyntax/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service.impl;
+
+import static core.basesyntax.model.FruitTransaction.Operation.BALANCE;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImplTest {
+    private static ParserService parserService;
+    private static List<String> dataList;
+
+    @BeforeAll
+    static void beforeAll() {
+        parserService = new ParserServiceImpl();
+        dataList = new ArrayList<>();
```

Comment:
`dataList` variable at class level is redundant because it has different values in every test, so you can introduce variable directly in test methods using `List.of()` method.